app/components/PercentageStats/index.tsx

```diff
@@ -87,7 +84,7 @@ function PercentageStats(props: Props) {
                 </>
             )}
             headingSize={headingSize}
-            headerDescription={headerDescription}
+            headerDescription={headerDescription || subHeading}
```

Comment:
Looks sus. @subinasr can you check all the places where PercentageStats is used?